File: 2023-2/exemplo-iterativa.py
import sys

# ler a altura do armário
h = int(input());

# pedir para Mochi testar a altura 1
print(1)
sys.stdout.flush();

# ler a resposta de Mochi
ans = input();

# responder qual é a altura máxima da qual o copo pode ser jogado sem quebrar
if ans == "Quebrou":
    print("! 0");
else:
    print("! 1");

File: 2023-2/exemplo-iterativa.cpp
#include <bits/stdc++.h>
 
using namespace std;
 
int32_t main() {
  ios::sync_with_stdio(false);
  cin.tie(NULL);

  // ler a altura do armário
  int h; cin >> h;
  string ans;

  // pedir para Mochi testar a altura 1
  cout << "1\n";
  cout.flush();

  // ler a resposta de Mochi
  cin >> ans;

  // responder qual é a altura máxima da qual o copo pode ser jogado sem quebrar
  if (ans == "Quebrou") cout << "! 0\n";
  else cout << "! 1\n";
 
  return 0;
}
